Elevation_Projects/My_Roster_EJN/server.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express')
+const app = express()
+const path = require('path')
+const urllib = require('urllib')
+app.use(express.static(path.join(__dirname, 'dist')))
+app.use(express.static(path.join(__dirname, 'node_modules')))
+
+
+const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756"
+}
+
+let data = []
+
+app.get('/teams', function (req, res) {
+    urllib.request('http://data.nba.net/10s/prod/v1/2018/players.json', function (err, response) {
+        data = JSON.parse(response.toString()).league.standard
+        res.end()
+    })
+})
+
+app.get('/teams/:teamName', function (req, res) {
+    console.log('hello');
+    let teamName = req.params.teamName
```

Comment:
you can deconstruct objects
`const { teamName } = req.params`

it is the same as doing: 
`const teamName = req.params.teamName`